frontend/encore/virtual-machine.rst

```diff
@@ -0,0 +1,106 @@
+Using Encore in a Virtual Machine
+=================================
+
+You may encounter some issues when using Encore in a virtual machine, like VirtualBox or VMWare.
+
+Fix watching issues
+-------------------
+
+When using a virtual machine, your project root directory is shared with the virtual machine with `NFS`_.
+This is really useful, but it introduces some issues with files watching.
+
+You must enable `polling`_ option to make it works:
+
+.. code-block:: javascript
+
+    // webpack.config.js
+
+    // ...
+
+    // will be applied for `encore dev --watch` and `encore dev-server` commands
+    Encore.configureWatchOptions(watchOptions => {
+        watchOptions.poll = 250; // check for changes every 250 ms
+    });
+
```

Comment:
Maybe also add a note here about the CLI equivalent (`--watch-poll`)?

Reply:
Yep why not, but it means that the user will have to pass `--watch-poll` two times in its [`package.json`](https://github.com/symfony/recipes/blob/master/symfony/webpack-encore-bundle/1.0/package.json#L10-L12) :confused: